src/components/Group/GroupEdit.vue

```diff
@@ -47,6 +47,7 @@
                 v-model="edit.description"
                 type="textarea"
                 :min-rows="3"
+                @keyup.ctrl.enter="maybeSave"
```

Comment:
There's another `textarea` above this one ("edit.publicDescription"), can you add the handler there as well?